.github/scripts/install-libsecp256k1.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -eux -o pipefail
+
+## The following script builds and installs libsecp256k1 to ~/.local/lib
+
+INSTALL_VERSION=5dcc6f8dbdb1850570919fc9942d22f728dbc0af
+
+if [[ "$(uname -s)" =~ ^MSYS_NT.* ]]; then
+    echo "This script is only meant to run on Windows under MSYS2"
+    exit 1
+fi
+
+if [ -f "$HOME/.local/lib/libsecp256k1.a" ]; then
+  echo "libsecp256k1 exists, exiting..."
+  exit 0
+fi
```

Comment:
are we expecting this line to ever be executed in our ci env?

Reply:
Also no, I'll drop these ones